sdk/typescript/src/signers/txn-data-serializers/txn-data-serializer.ts

```diff
@@ -156,6 +156,11 @@ export type SignedTransaction = {
   signature: SerializedSignature;
 };
 
+export type SignedMessage = {
```

Comment:
I'd suggest renaming this to PersonalMessage as this is the standard term (and avoid confusion with a transaction message (many times in crypto we generically use the term message to define any message type).

Reply:
I think in the SDK context it's fairly unambiguous, as we have `signMessage` + `SignedMessage` and `signTransaction` + `SignedTransaction`.